server/channels/api4/user_test.go

```diff
@@ -2300,7 +2360,8 @@ func TestUpdateUserAuth(t *testing.T) {
 	_, err = th.App.Srv().Store().User().VerifyEmail(user2.Id, user2.Email)
 	require.NoError(t, err)
 
-	th.SystemAdminClient.Login(context.Background(), user2.Email, "passwd1")
+	_, _, err = th.SystemAdminClient.Login(context.Background(), user2.Email, "passwd1")
```

Comment:
The password is actually `Pa$$word11`. See https://github.com/mattermost/mattermost/blob/68404f2f335d0ccd38db8ba0e7156a2840e98dab/server/channels/api4/apitestlib.go#L624

The check in L2369 is also wrong. It should check the error, not the response.